module/documents/actor.mjs

```diff
@@ -99,6 +102,8 @@ export default class ActorEd extends Actor {
     }
   }
 
+  
```

Comment:
Remove empty lines.
Also: `#processHistoryEarned` is not implemented. This should only be committed if it's fixed that it is going to be used. If yes, then it should be documented why it is not implemented yet.